src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -214,12 +213,13 @@ public Task<ExecutionResult> ResetAsync(bool initialize)
                     _window._currentLanguageBuffer = null;
                 }
 
-                // replace the task being interrupted by a "reset" task:
                 State = State.Resetting;
-                _currentTask = _window._evaluator.ResetAsync(initialize);
-                _currentTask.ContinueWith(FinishExecute, _uiScheduler);
+                var executionResult = await _window._evaluator.ResetAsync(initialize).ConfigureAwait(true);
```

Comment:
While it seems like overkill to assert that ConfigureAwait worked, I'm always willing to add more asserts. :)